Miki.Framework/Miki.Framework.csproj

```diff
@@ -18,10 +18,11 @@
     <RepositoryUrl>https://github.com/mikibot/miki.framework</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <Copyright>Velddev</Copyright>
+    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>latestmajor</LangVersion>
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Prod|AnyCPU'">
```

Comment:
This would fail the build if we're building the configuration `Prod`.
Both the PropertyGroup of Debug and Prod can probably be removed, since we're always using C# 8. If you're wondering how this end up in here: Visual Studio auto generates these when you change the C# version in its solution settings.